db/seeds.rb

```diff
@@ -8,3 +8,234 @@
 
 # Attach images
 # equipment.image.attach io: File.open("spec/assets/test.png"), filename: "test.png", content_type: "image/png"
+
+admin = User.create({
+  name: FFaker::Name.name,
+  username: FFaker::Internet.user_name,
+  email: FFaker::Internet.email,
+  password: "test1234",
+  role: 'admin'
+})
+
+10.times.map do
+    User.create({
+    name: FFaker::Name.name,
+    username: FFaker::Internet.user_name,
+    email: FFaker::Internet.email,
+    password: "test1234",
+    role: 'user'
+  })
+end
+
+three_d_printer_ender = Equipment.new({
+  title: "3D Printer Ender",
+  description: "An open-source 3d printer with amazing printing precision and affordable price, quite the best 3d printer for beginners. Allow users to improve their printer upon the source code and share the changes within the community, resulting in an improved experience for all.",
+  review: "My co-worker Ali has one of these. He says it looks towering.",
```

Comment:
It would have been a good idea to add user reviews 🤔. But actually, this field corresponds to an URL link to a youtube review video

Reply:
Damn bro, aight, I'll make those changes, thanks!

Reply:
No problem! Thanks to you